### PR TITLE
Publish release 1.4.3.

### PR DESCRIPTION
**Pull request by Tatsinnit**
Thank you so much all for the contributions and as always helping. 🙏❤️☕️

This release incorporate following changes:

* Retina integration with AKS Cluster via VsCode Extension.
* In-house download replacement implementation.
* Show-properties page k8s version is now available with deprecated warning.
* Dependabot updates and bumps.

Thank you all for continued awesomeness, discussion and improvements, @sprab, @hsubramanianaks, @peterbom, @rbtr, @vakalapa, ❤️🙏 cc and fyi: @gambtho + @neha170 

VSIX to play around: 
[vscode-aks-tools-1.4.3-test.vsix.zip](https://github.com/Azure/vscode-aks-tools/files/15069530/vscode-aks-tools-1.4.3-test.vsix.zip)

Kind regards,

### CODE REVIEW

File: docs/book/src/features/retina-capture.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Run Retina Distributed Capture from AKS Cluster Linux Nodes
+
+### Run Retina Capture
+
+Right click on your AKS cluster and select **Troubleshoot Network Health** and then click on **Run Retina Capture** to capture logs like iptables-rules, [ip-resrouces.txt and other key disctributed captures form this azure networking tool](https://retina.sh/docs/captures/cli#file-and-directory-structure-inside-the-tarball) for any Linux node and download them to your local machine with ease.
```
Comment (peterbom) [inline docs/book/src/features/retina-capture.md:5]: spelling: 'disctributed'

**Review by peterbom — APPROVED** [review]

File: docs/book/src/features/retina-capture.md (hunk 1 above)
Comment (Tatsinnit) [reply docs/book/src/features/retina-capture.md]: Thank you, fixed.

**Review by peterbom — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
 # Change Log
 
+## [1.4.3]
+
+* Retina integration with VsCode.
+* In-house download replacement implementation.
+* Show-properties page k8s version is now available with deprecated warning.
+* Dependabot updates and bumps.
```
Comment (hsubramanianaks) [inline CHANGELOG.md:8]: nit: Dependent Bot updates*

**Review by hsubramanianaks — APPROVED** [review]
lgtm, small nit change.

File: CHANGELOG.md (hunk 2 above)
Comment (Tatsinnit) [reply CHANGELOG.md:8]: :) its not spelling mistake it shout the real name of the bot :) 

Screenshot 2024-04-23 at 12 08 36 PM

**Review by sprab — APPROVED** [review]
This works as expected. I have tested Retina capture along with other previous features.  Everything works.

**Pull request merged**